EasyLearn/app/assets/stylesheets/styles.css

```diff
@@ -0,0 +1,145 @@
+.contenido{
+  background-color: #282828;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.nav{
+  align-items: center;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  width: 80%;
+}
+ 
+.nav__items{
+  align-items: center;
+  display: flex;
+  font-size: 25px;
+  gap: 2rem;
+}
+ 
+.dropdown > .btn{
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.main__content{
+  border-top: 1px solid orange;
+  background-color: #282828;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.4rem;
+  height: 99.3vh;
+  width: 100%;
+}
+
+.aside__profile{
+  display: flex;
+  align-items: center;
+  padding-top: 4rem;
+  padding-left: 1rem;
+  justify-content: center;
+}
+.aside__picture{
+  width: 35%;
+  
+}
```

Comment:
Please, respect the blank spaces between each block of code and delete the extra blank lines